### ISSUE TITLE
[Accessibility] Revert bcfbe99b73e69198fd303e5b4f0ac5be2437e6c6 aXe - fix missing localization for aria-label strings

### ISSUE THREAD

**Issue by majornista** [adobe/coral-spectrum#20]
### Expected Behavior
Coral-Spectrum should not add non-meaningful label strings to components in order to pass aXe accessibility unit tests. For example: 
```html
<label class="_coral-Checkbox-label" handle="labelWrapper" for="{{this.input.id}}">
  <span class="u-coral-screenReaderOnly" handle="screenReaderOnly">{{data.i18n.get('Select')}}</span>
</label>
```
From https://github.com/adobe/coral-spectrum/commit/bcfbe99b73e69198fd303e5b4f0ac5be2437e6c6#diff-931723f1f811c434792fe675311bcc82R9

If a control is not labeled, it should throw an error rather than mask accessibility problems with generic labels, e.g. "Select" for a checkbox. 

Instead, we should update unit tests and examples to include labels to avoid accessibility failures when running `gulp axe`.

### Actual Behavior
Controls like Checkbox and List include non-meaningful default labels like "Select" or "List", that trick the aXe accessibility testing tool into thinking all controls are labeled. In production, this masks accessibility failures by incorrectly labeling controls.

### Reproduce Scenario (including but not limited to)
Observe this simple example of a checkbox without a label:

```html
<coral-checkbox />
```

A hidden default accessibility label "Select" has been added to the control. An automated accessibility testing tool, like aXe, will fail to identify this as a problem.

#### Steps to Reproduce

1. Open  https://opensource.adobe.com/coral-spectrum/dist/playground/#?livereload=true&screen=vertical&code=MTczLDE0NCw2MSwxNCwxOTQsNDgsMTIsMTMzLDExOSw3OCwxNywxNzgsMTgzLDIyMiw4MSwyMTgsNSwxNTIsOTcsOTYsOTcsNzYsMjksMjExLDIwLDIxMCwzOCwxMzgsOTMsMTM3LDIyMiwxNTgsMjEwLDEzNiwyNyw0OCwyNDksNzEsMjQ2LDI0NywyNTIsMTA4LDI0NiwxNjcsMjAzLDI0MSwxMTgsMTkxLDE1OCwxNDksMTUxLDQ5LDE4MCw1OSwyNDMsMTEsMTAwLDkzLDE4Nyw4MywyMDIsMTMyLDk3LDEyMiwxNjksNzYsMTYxLDIwOSw0NCw3NSwzMiwyNDYsNjgsMTYyLDE0OSwyMDcsMjQ0LDEwNCwxODAsMjMsNzMsMTI0LDAsMTM2LDEzNywzOCwxNDIsMTE1LDcwLDE3MCwxNzMsMTM5LDI5LDIxMywyNCw3MSwxOTIsMTUyLDEwOSwxNjgsNTYsMTcsNzQsMTU4LDcxLDExMiwzLDExLDE2NCw5NiwxNTEsNjIsMTk5LDEyMSwxMTQsMTY1LDcwLDIzMCw1MCw4NywxNzUsMTUzLDEzNCw3NywxNDQsNDksMTUsNzMsMjAsMTAzLDI1MiwxMzEsMTkyLDI0MywxOTksMTI3LDE3OCwxMTAsMTMsMjAsMjQ2LDEwNiwxNSwxMzgsNjMsMjExLDY5LDE4Myw0MCwxMiwxNTAsMTg1LDIwOSwxMzMsODcsMTMzLDE2MSwyNDcsMTYyLDE4Myw4Myw3NCw3LDYxLDIyNSwxNzEsMTM5LDExMSw1LDIyMywxOTcsMjM5LDE5OCw2LDIxNiwyNTQsMjQ0LDE=&
2. Inspect the rendered checkbox with Web Inspector
3. Under Web Inspector > Accessibility panel note that the checkbox without a visible is labeled "Select"

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)
All

#### Coral Spectrum version
4.0.1

#### Sample Code that illustrates the problem (use the Playground if possible)
https://opensource.adobe.com/coral-spectrum/dist/playground/#?livereload=true&screen=vertical&code=MTczLDE0NCw2MSwxNCwxOTQsNDgsMTIsMTMzLDExOSw3OCwxNywxNzgsMTgzLDIyMiw4MSwyMTgsNSwxNTIsOTcsOTYsOTcsNzYsMjksMjExLDIwLDIxMCwzOCwxMzgsOTMsMTM3LDIyMiwxNTgsMjEwLDEzNiwyNyw0OCwyNDksNzEsMjQ2LDI0NywyNTIsMTA4LDI0NiwxNjcsMjAzLDI0MSwxMTgsMTkxLDE1OCwxNDksMTUxLDQ5LDE4MCw1OSwyNDMsMTEsMTAwLDkzLDE4Nyw4MywyMDIsMTMyLDk3LDEyMiwxNjksNzYsMTYxLDIwOSw0NCw3NSwzMiwyNDYsNjgsMTYyLDE0OSwyMDcsMjQ0LDEwNCwxODAsMjMsNzMsMTI0LDAsMTM2LDEzNywzOCwxNDIsMTE1LDcwLDE3MCwxNzMsMTM5LDI5LDIxMywyNCw3MSwxOTIsMTUyLDEwOSwxNjgsNTYsMTcsNzQsMTU4LDcxLDExMiwzLDExLDE2NCw5NiwxNTEsNjIsMTk5LDEyMSwxMTQsMTY1LDcwLDIzMCw1MCw4NywxNzUsMTUzLDEzNCw3NywxNDQsNDksMTUsNzMsMjAsMTAzLDI1MiwxMzEsMTkyLDI0MywxOTksMTI3LDE3OCwxMTAsMTMsMjAsMjQ2LDEwNiwxNSwxMzgsNjMsMjExLDY5LDE4Myw0MCwxMiwxNTAsMTg1LDIwOSwxMzMsODcsMTMzLDE2MSwyNDcsMTYyLDE4Myw4Myw3NCw3LDYxLDIyNSwxNzEsMTM5LDExMSw1LDIyMywxOTcsMjM5LDE5OCw2LDIxNiwyNTQsMjQ0LDE=&

**Comment by icaraps** [comment]
Affected components are: Checkbox, Drawer, List, Radio, Switch

**Comment by icaraps** [comment]
Closing in favor of https://github.com/adobe/coral-spectrum/commit/947b137944f57fae9373fb680b9a96634e5403d0
